utils/blog/api-helpers.ts

```diff
@@ -0,0 +1,28 @@
+export function getStrapiURL(path = '') {
+	if (!process.env.NEXT_PUBLIC_STRAPI_API_URL) {
+		console.error(
+			`[getStrapiURL] NEXT_PUBLIC_STRAPI_API_URL env variable is not set, unable to get strapi URL`);
+		throw new Error(`NEXT_PUBLIC_STRAPI_API_URL env variable is not set, unable to get strapi URL`);
+	}
+	return `https://${process.env.NEXT_PUBLIC_STRAPI_API_URL}${path}`;
+}
+
+export function getStrapiMedia(url: string | null) {
+	if (url == null) {
+		return null;
+	}
+
+	// Return the full URL if the media is hosted on an external provider
+	if (url.startsWith('http') || url.startsWith('//')) {
+		return url;
+	}
+
+	// Otherwise prepend the URL path with the Strapi URL
+	return `${getStrapiURL()}${url}`;
```

Comment:
`getStrapiURL` function takes `url` as a parameter, but you are not using it that way. 

```suggestion
	return getStrapiURL(url);
```